doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -215,7 +220,9 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`bt_fast_pair_readme` library:
+
+  * Added option to set the flag for the hide UI indication in the Fast Pair not discoverable advertising data.
```

Comment:
`Extended API to allow setting the flag ...`

Reply:
`option` may seem like reference to Kconfig option which could be a bit confusing